RCS file: /pack/cvsroots/wxwindows/wxWindows/docs/latex/wx/toolbar.tex,v
retrieving revision 1.39
diff -b -u -2 -r1.39 toolbar.tex
--- toolbar.tex	2004/05/04 08:25:50	1.39
+++ toolbar.tex	2004/05/07 12:49:46
@@ -52,4 +52,18 @@
 \end{verbatim}
 
+If you wish to use 32-bit images (which include an alpha channel for transparency)
+use:
+
+\begin{verbatim}
+  wxSystemOptions::SetOption(wxT("msw.remap"), 2);
+\end{verbatim}
+
+then colour remapping is switched of, and a transparent background used. But only
+use this option under Windows XP with true colour:
+
+\begin{verbatim}
+  (wxTheApp->GetComCtl32Version() >= 600 && ::wxDisplayDepth() >= 32)
+\end{verbatim}
+
 \wxheading{Window styles}

 	  	 
